qualtran/bloqs/rotations/phase_gradient.py

```diff
@@ -476,3 +533,18 @@ def add_my_tensors(
         _add_my_tensors_from_gate(
             self, self.signature, self.pretty_name(), tn, tag, incoming=incoming, outgoing=outgoing
         )
+
+
+@bloq_example
+def _add_scaled_val_into_phase_reg() -> AddScaledValIntoPhaseReg:
+    add_scaled_val_into_phase_reg = AddScaledValIntoPhaseReg(
+        QFxp(2, 2), phase_bitsize=2, gamma=2, gamma_dtype=QFxp(2, 2)
+    )
+    return add_scaled_val_into_phase_reg
+
+
+_ADD_SCALED_VAL_INTO_PHASE_REG_DOC = BloqDocSpec(
+    bloq_cls=AddScaledValIntoPhaseReg,
+    import_line='from qualtran.bloqs.rotations.phase_gradient import AddIntoPhaseGrad',
```

Comment:
!!!

Reply:
Just fixed them, I was not aware they were put in the rotations module instead of their own. I still might have to figure out the failing checks before this is fully ready.